### PR TITLE
recipes-support: mavlink-router: Enable flight stack logging

### PR DESCRIPTION
**Pull request by zehortigoza**

### CODE REVIEW

File: recipes-support/mavlink-router/mavlink-router_0.1.bb [hunk 1]
```diff
@@ -23,6 +23,8 @@ do_install_append () {
 	install -d ${D}${sysconfdir}/mavlink-router/config.d
 	install -m 0755 ${WORKDIR}/main.conf ${D}${sysconfdir}/mavlink-router/main.conf
 
+	install -d ${D}/home/root/logs
```
Comment (lucasdemarchi) [inline recipes-support/mavlink-router/mavlink-router_0.1.bb]: @edersondisouza we can do this here, but mavlink-router should create the dir itself. You can use the mkdir_p function from kmod for that

Comment (lucasdemarchi) [inline recipes-support/mavlink-router/mavlink-router_0.1.bb]: See https://github.com/lucasdemarchi/kmod/blob/master/shared/util.c#L397

**Comment by lucasdemarchi** [comment]
> Should rather be /var/mavlink-router/logs

I changed this to /var/lib/mavlink-router/logs and applied (but didn't push it yet, doing some tests)

**Comment by lucasdemarchi** [comment]
continued on #77

File: recipes-support/mavlink-router/mavlink-router_0.1.bb (hunk 1 above)
Comment (edersondisouza) [inline recipes-support/mavlink-router/mavlink-router_0.1.bb]: https://github.com/01org/mavlink-router/issues/58 to keep track

**Pull request closed without merging**